pkg/virt-controller/services/template.go

```diff
@@ -1004,9 +998,6 @@ func (t *templateService) RenderLaunchManifest(vmi *v1.VirtualMachineInstance) (
 			Subdomain: vmi.Spec.Subdomain,
 			SecurityContext: &k8sv1.PodSecurityContext{
 				RunAsUser: &userId,
-				SELinuxOptions: &k8sv1.SELinuxOptions{
-					Type: selinuxLauncherType,
-				},
```

Comment:
I would still keep this. Just would set the default to `container_t`. Just in case some systems will need to use a different label. I've seen that there was a new `container_kvm_t` introduced recently.

Reply:
Well, I removed it on purpose.
We don't manage container_t. The reason we want virt-launcher to run as container_t is because that's what k8s wants to run it as, to be able to isolate it with SELinux.
If tomorrow k8s decides to use another type to label and secure containers, virt-launcher should follow.

Reply:
I'm not sure if that applies to the virt-launcher case. It's still special since we run virtualization components in it and should have room to set a different label.

Reply:
So this ended up being more than a philosophical question, since I just confirmed that forcing container_t leads to different behaviors than not forcing a type and letting it default to container_t.
I understand the need to leave room to set a different label, and I'll rework the code accordingly.